src/components/CrossDataSection.tsx

```diff
@@ -45,7 +46,7 @@ export const CrossDataSection = ({ dataProviderId, dataProviderName, navigation
   const variables = {
     dataProviderId,
     orderEventRecords: 'listDate_ASC',
-    limit: 3
+    limit: 4
```

Comment:
why 4 now?

Reply:
because i will not know if there are more than 3 and need to show the button, if i only fetch at most 3 here.

Reply:
ok, sure.
could be written somewhere.